docs/index.rst

```diff
@@ -10,15 +10,13 @@ SOSS Documentation
     :alt: eProsima
     :target: http://www.eprosima.com/
 
-*SOSS* is intended to be a *System-Of-Systems Syntesizer*, That is, if you have a pair of complex systems, A and B,
-and you want to combine them to create a larger, even more complex system, the goal is for SOSS to make that task
-easier.
+*SOSS* is a *System-Of-Systems Syntesizer* that allows communication among an arbitrary number of protocols that speak different languages.
+If one has for instance a pair of complex systems, A and B, and wills to combine them to create a larger, even more complex system, SOSS can be chosen as an intermediate message-passing protocol that, by speaking a common language, can mediate this integration. The A<->B bridge is configured with a text file, through which the user can provide a mapping between the topics and services on System A's middleware and those on System B.
```

Comment:
Split long lines (try to adjust them into 120 characters).

Reply:
ok. see if you like it.

Reply:
The line is still too long. I don't mean by punctuation but literally the length of the line in the editor. You must follow the coding style (max. 120 chars per line).

Reply:
ok. done everywhere.